README.md

```diff
@@ -182,41 +182,6 @@ about Keptn on the
 Videos that refer to the "Keptn Lifecycle Controller"
 are relevant for the Keptn project.
 
-## Architecture
-
-Keptn is composed of the following components:
-
-- Keptn Lifecycle Operator
-- Keptn Scheduler
-
-The Keptn Lifecycle Operator contains several controllers for Keptn CRDs
```

Comment:
Shouldn't the information from here down be integrated into the components/lifecycle-operator page before it is deleted here?

Reply:
IMO this information here is too detailed to be maintainable and if we already have more general information like this in the components section.

Reply:
Okay.  I'm not sure I agree -- but I am inclined to put much more technical background into the docs than you are.  So I'll mark this as resolved and we'll see if we have user questions that arise that need the deeper info.